lib/wasi/Cargo.toml

```diff
@@ -10,9 +10,6 @@ license = "MIT"
 readme = "README.md"
 edition = "2018"
 
-[lib]
-crate-type = ["cdylib", "rlib"]
```

Comment:
I believe this is needed for wasm-bindgen / wasmer-js (wasi)

Reply:
This is necessary for the lto though. Do you know what command/build fails for wasm-bindgen/wasmer-js if this is removed?

Reply:
We can add the crate-type as a special case.

Reply:
**Runtime tests - JS** have passed by the way.